src/components/Definition.jsx

```diff
@@ -0,0 +1,13 @@
+export default function Definition({ children }) {
+  return (
+    <div className="cmp-definition">
+      <h2 className="cmp-definition__heading">
+        Definition
+        <img src="circle.svg" className="cmp-definition__heading-circle" alt="Sketched circle, drawn around header" />
```

Comment:
Would this work better as a background image on `cmp-definition__heading`, since it's not a part of the content and is purely decorative?

If this is kept as an `<img>`, I would use an empty `alt=""` attribute so that screen readers will skip reading this, since it's not necessary for understanding the content.